test/internet/test-dns-setserver-in-callback-of-resolve4.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const common = require('../common');
+const dns = require('dns');
+
+dns.resolve4('google.com', common.mustCall(function(/* err, nameServers */) {
+  // do not care about `err` and `nameServers`,
+  // both failed and succeeded would be OK.
+  //
+  // just to test crash
+
+  dns.setServers([ '8.8.8.8' ]);
+
+  // the test case shouldn't crash here
```

Comment:
These comments can probably be dropped, but if you want to keep them, could you move them to the top like [this](https://github.com/nodejs/node/blob/5debcceafcdd73035d840f53deb931925691a3ab/test/known_issues/test-cwd-enoent-file.js#L2-L4). Note the `Refs:` notation.